mountpoint-s3/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## Unreleased
 
+### Other changes
+
 * Allow changing log level dynamically with `USR2` signal. See [Changing logging verbosity at runtime](https://github.com/awslabs/mountpoint-s3/blob/main/doc/LOGGING.md#changing-logging-verbosity-at-runtime) for more details. ([#1367](https://github.com/awslabs/mountpoint-s3/pull/1367))
+* Enable caching of credentials when `--profile` CLI argument is used. ([#1398](https://github.com/awslabs/mountpoint-s3/pull/1398))
```

Comment:
nit: I'd consider this a fix rather than a feature, so I'd probably word it more like "Fix missing caching of credentials when using `--profile` CLI argument"